.github/workflows/build.yml

```diff
@@ -0,0 +1,33 @@
+
```

Comment:
We have not been using github workflows for building on this repo. I realize that we need a way to get macos build agents, but I think we can do that in Azure Devops? Also tricky to sync this so we only build/publish when all succeed.

Reply:
I did try Azure's macOS build agents in the beginning - but they just did not work at all.  The pipelines failed early in the build configuration process on those machines.  With the next patch to this PR, I can again set up to test build there, and see how that goes now.